src/style.css

```diff
@@ -1,3 +1,88 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:500&display=swap';
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --main-color: #000;
+  --blue-color: #00acdc;
+
+  font-size: 12px;
+  font-family: Roboto, serif;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  list-style: none;
+}
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). It's better to style elements using classes to avoid conflicts and improve reusability.